utm.yaml

```diff
@@ -983,9 +987,139 @@ components:
+
+        Reference: https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e32-178-1
+      required:
+      - uas_serial_number
+      - operation_mode
+      - operation_category
+      - uas_class
+      - identification_technology
+      - connectivity_methods
+      - endurance
+      - operator_id
+      type: object
+      properties:
+        uas_serial_number:
+          description: |-
+            Unique serial number of the unmanned aircraft or, if the unmanned aircraft
+            is privately built, the unique serial number of the add-on.
+            This is expressed in the ANSI/CTA-2063 Physical Serial Number format.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 1.
+          type: string
+        operation_mode:
+          description: |-
+            Mode of operation.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 2.
+          type: string
+          enum:
+          - vlos
+          - bvlos
+        operation_category:
+          description: |-
+            Category of UAS operation (‘open’, ‘specific’, ‘certified’) as defined in COMMISSION
+            DELEGATED REGULATION (EU) 2019/945.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 4.
+          type: string
+          enum:
+          - open
+          - specific
+          - certified
+        uas_class:
+          description: |-
+            UAS aircraft class as defined in COMMISSION DELEGATED REGULATION (EU) 2019/945 (C0 to C4)
+            and COMMISSION DELEGATED REGULATION (EU) 2020/1058 (C5 and C6).
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 4.
+          type: string
+          enum:
+          - C0
+          - C1
+          - C2
+          - C3
+          - C4
+          - C5
+          - C6
+          - other
+        uas_type_certificate:
+          description: |-
+            Provisional field. Not applicable as of September 2021.
+
+            Required only if `uas_class` is set to `other` by ANNEX IV of COMMISSION IMPLEMENTING
+            REGULATION (EU) 2021/664, paragraph 4.
+          type: string
+        # TODO: clarify options
+        identification_technologies:
+          description: |-
+            Technology used to identify the UAS.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 6.
+          type: array
+          items:
+            type: string
+          example: ["netrid", "ads-b"]
+        # TODO: clarify options
+        connectivity_methods:
+          description: |-
+            Connectivity methods.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 7.
+          type: array
+          items:
+            type: string
+        endurance:
```

Comment:
Could we name this field `endurance_minutes` to make absolutely clear what the units are?